GOVERNANCE.md

```diff
@@ -1,66 +1,119 @@
+The WebdriverIO Project wants as much as possible to operate using procedures that are fair, open, inviting, and ultimately good for the community. For that reason we find it valuable to codify some of the ways that the Project goes about its day-to-day business. We want to make sure that no matter who you are, you have the opportunity to contribute to WebdriverIO. We want to make sure that no corporation can exert undue influence on the community or hold the Project hostage. And likewise we want to make sure that corporations which benefit from WebdriverIO are also incentivized to give back. This document describes how various types of contributors work within the WebdriverIO project.
 
-### The Technical Committee
+## Roles and Responsibilities
 
-The project is officially led by a Technical Committee (TC), currently consisting of:
+### Users
 
-* [@abjerstedt](https://github.com/abjerstedt)
-* [@bromann](https://github.com/christian-bromann)
-* [@willbrock](https://github.com/willbrock)
+Users are community members who have a need for the project. Anyone can be a User; there are no special requirements. Common User contributions include evangelizing the project (e.g., display a link on a website and raise awareness through word-of-mouth), informing developers of strengths and weaknesses from a new user perspective, or providing moral support (a "thank you" goes a long way).
 
-They all represent different companies and different types of contribution to the WebdriverIO Project. The TC is responsible for high-level decisions (like [release a new version](/CONTRIBUTING.md#release-new-version), establishing a feature roadmap, accepting sponsorship, organizing conferences, etc...), adjudicating conflicts between members of the community, and making changes to the project's governance processes.
+Users who continue to engage with the project and its community will often become more and more involved. Such Users may find themselves becoming Contributors, as described in the next section.
 
-* Membership in the TC is granted by unanimous vote of current TC members.
-* TC members can step down at any time.
-* No more than 1/3 of TC members may belong to the same company.
-* Changes to project governance require unanimous vote of the TC.
+### Contributors
+
+Contributors are community members who contribute in concrete ways to the project, most often in the form of code and/or documentation. Anyone can become a Contributor, and contributions can take many forms. There is no expectation of commitment to the project, no specific skill requirements, and no selection process.
+
+Contributors have read-only access to source code and so submit changes via pull requests. Contributor pull requests have their contribution reviewed and merged by a TSC member. TSC members and Committers work with Contributors to review their code and prepare it for merging.
```

Comment:
Contributors have read-only access to source code and to submit changes via pull requests.

Reply:
Updating `so` to `to` and then I think the pr is good to go.